pkg/dbadmin/mysqladmin/admin.go

```diff
@@ -213,6 +196,83 @@ func (mdba *MySQLDbAdmin) GetSchemaVersion() (string, error) {
 	return version, nil
 }
 
+// GetTableSizeEstimates implements DbAdmin
+func (mdba *MySQLDbAdmin) GetTableSizeEstimates(tableNames []dbadmin.TableName) (map[dbadmin.TableName]uint64, error) {
+	estimates := make(map[dbadmin.TableName]uint64)
+
+	if len(tableNames) == 0 {
+		return estimates, nil
+	}
+
+	query, args, err := sqlx.In("SELECT TABLE_NAME, TABLE_ROWS FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME IN (?);", tableNames)
+	if err != nil {
+		return estimates, fmt.Errorf("Unable to prepare query to load table size estimates: %w", err)
+	}
+
+	var results []struct {
+		TableName string `db:"TABLE_NAME"`
+		TableRows uint64 `db:"TABLE_ROWS"`
+	}
+	query = mdba.handle.Rebind(query)
```

Comment:
move this up to the call to `In()` since it's logically coupled to that call and not the `Select()`